_pages/plugins/ultrack.md

```diff
@@ -0,0 +1,109 @@
+4. Now you should be able to start the connection with the Ultrack software by clicking the **Start Ultrack Server** 
+   button. The plugin will update its GUI to show the available tracking options, as shown in the figure below.
+
+[![Ultrack GUI with server started](/media/plugins/ultrack/01_gui.png)](/media/plugins/ultrack/01_gui.png)
+
+4. The user now should select the desired tracking workflow in this section:
+    
+[![Ultrack GUI with tracking workflows](/media/plugins/ultrack/03_workflows.png)](/media/plugins/ultrack/03_workflows.png)
+    
+in which the user can select the desired tracking workflow. The options are:
+ - **Auto Detection From Image**: which provides a way to track cells using solely the image as input;
+ - **Manual Detection**: which is suitable for tracking cells using your custom **edge** detection and **foreground** 
+   segmentation algorithms;
+ - and **Auto Detection From Label**: which is suitable for tracking cells using a pre-segmented image from your favorite 
+   segmentation software, such as [Stardist](https://github.com/stardist/stardist), 
+   [Cellpose](https://github.com/MouseLand/cellpose), 
+   [MicroSAM](https://github.com/computational-cell-analytics/micro-sam) or any other.
+
+5. (Optional) After selecting your desired workflow, you can tune the tracking parameters in the **Settings** section. 
+   The parameters are workflow-dependent and the user can 
+   [check here](https://github.com/royerlab/ultrack-dev/blob/main/ultrack/config/README.md) for more information about
+   the parameters.
+
+[![Ultrack GUI with tracking options](/media/plugins/ultrack/02_options.png)](/media/plugins/ultrack/02_options.png)
+
+6. After setting the parameters, the user can click the **Select Images** button to select the image to be tracked. 
+   Is worth mentioning that some workflows require more than one image to be selected, such as the **Manual Detection**.
+
+[![Ultrack GUI with image opening](/media/plugins/ultrack/04_image_opening.png)](/media/plugins/ultrack/04_image_opening.png)
+[![Ultrack GUI with image selector](/media/plugins/ultrack/05_image_selection.png)](/media/plugins/ultrack/05_image_selection.png)
+
+7. Finally, the user can click the **Run** button to start the tracking process. The user can follow the tracking 
+   process in the **Ultrack Log** section. 
+
+[![Ultrack GUI with tracking progress](/media/plugins/ultrack/06_run.png)](/media/plugins/ultrack/06_run.png)
+
+8. To visualize the tracking results, the user can click the **View Tracks** button. Then, it will be requested a 
+   particular image to bind the tracks with. In the end, The tracking results will be shown using the 
+   [TrackMate](https://imagej.net/plugins/trackmate/) plugin. 
+
+[![Ultrack GUI with track image selection](/media/plugins/ultrack/07_open_tracks.png)](/media/plugins/ultrack/07_open_tracks.png)
+[![Tracks Viewer it Trackmate](/media/plugins/ultrack/08_trackmate.png)](/media/plugins/ultrack/08_trackmate.png)
+
```

Comment:
Please add  an acknowledgments section and thank trackmate and other Fiji packages you depend on.